bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-drop",
-  "version": "1.0.0",
-  "framework": ">=5.3",
+  "version": "1.0.1",
+  "framework": ">=5.5",
```

Comment:
Did we miss something 5.5-y in the existing code?

Reply:
probably not, we just seem to have been making everything that uses ES6 have a FW dependency of 5.5+ - perhaps unnecessarily but I sort of feel like v5.5 was the first release where ES6 classes were properly confirmed as working.

I was tempted to bump it to 5.8 and change from `require` to `import` but that seemed a bit OTT for this PR